### DISCUSSION TITLE
Status query about CVE-2010-0928

### DISCUSSION THREAD

**Discussion by sebastianas** [openssl/openssl#24540] [Q&A] [answered]
The Debian security tracer has CVE-2010-0928 assigned to OpenSSL and is marked as "open". 
The entry contains a link to the paper http://www.eecs.umich.edu/~valeria/research/publications/DATE10RSA.pdf and there is additional info line
> impractical right now, but the openssl developers are working on a fix just in case

Both (the link and the line) has been added in 2010 and I have no additional notes.
Could the project please comment if this still affects openssl?

Sebastian

**Comment by nhorman** [comment]
From what I can see, this is still a potential attack vector, yes.  However:

1) While openssl does not appear to validate signatures during the signing process within the library, there is no reason that a consuming application could not write code to preform a signature sequence (EVP_DigestSignInit/Update/Final) in the same function as a signature verification (EVP_DigestVerifyInit/Update/Final), which would mitigate the problem

2) Multipe references to this issue indicate that an attack of this type requires that the attacker have direct access to the device in question, to the point that they can manipulate device voltage, making this a side channel attack that is not considered likely in the wild.

**Comment by t8m** [comment, marked as answer]
Fault injection based attacks are not within our threat model according to our security policy. https://www.openssl.org/policies/general/security-policy.html

**Reply by sebastianas** [reply]
Okay. That means the openssl developers are not working on a solution and this is not considers a threat. I think that works.
Thank you
